modules/remove-central-attached-pv-operator.adoc

```diff
@@ -3,7 +3,7 @@
 // * upgrading/upgrade-operator.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="remove-central-attached-pv-operator_{context}"]
-= Remove Central-attached PV using the {product-title-short} Operator
+= Remove Central-attached PV using the {product-title-short} Operator (version 4.1 and later)
```

Comment:
Ideally, this would be rewritten to exclude the parens. I don't think this is a total prohibition on it, but ISG generally allows them only as a last resort.

Reply:
It makes for a long heading but I rewrote.